OpenSim/Simulation/Model/Frame.h

```diff
@@ -193,6 +206,41 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Frame, ModelComponent);
     */
     SimTK::Vec3 findLocationInAnotherFrame(const SimTK::State& state, 
                     const SimTK::Vec3& point_F, const Frame& otherFrame) const;
+
+    /**
+    Take a point located and expressed in this frame (F) and determine
+    its location expressed in Ground (G). This method is equivalent to
+    `findLocationInAnotherFrame()` where `otherFrame` is always Ground.
+
+    @param state       The state of the model.
+    @param point_F     The point to be re-expressed.
+    @return point_G    The re-expression of the point in Ground.
+    */
+    SimTK::Vec3 findLocationInGround(const SimTK::State& state,
+                    const SimTK::Vec3& point_F) const;
+
+    /**
+    Take a point located and expressed in this frame (F) and determine
+    its velocity expressed in Ground (G).
+
+    @param state       The state of the model.
+    @param point_F     The point to be re-expressed.
+    @return point_G    The re-expression of the point in Ground.
+    */
+    SimTK::Vec3 findVelocityInGround(const SimTK::State& state,
```

Comment:
I agree that renaming might be good, but I think I prefer:

- SpatialVec or Vec6: getSpatialVelocityInGround()
- SpatialVec[0]: getAngularVelocityInGround()
- SpatialVec[1]: getVelocityInGround()

Therefore, I think the name of this particular method is good.